language/English/strings.po

```diff
@@ -200,6 +200,10 @@ msgctxt "#745"
 msgid "Set to ON to enable the cron daemon"
 msgstr ""
 
+msgctxt "#746"
+msgid "Set the password for SSH root"
```

Comment:
"Set the SSH password" .. because we only have one user, so use simpler wording

Reply:
sounds good